### PR TITLE
Gp2 1575 case study reordering renaming

### PR DESCRIPTION
**Pull request by webbyfox**
This changeset rename some of case study fields and tweak layout

### Workflow

- [x] Ticket exists in Jira https://uktrade.atlassian.net/browse/1575
- [x] Jira ticket has the correct status.
- [x] [Changelog](CHANGELOG.md) entry added.
- [x] This PR can be merged by reviewers. (If unticked, please leave for the author to merge)

image

### CODE REVIEW

**Review by dahfool — APPROVED** [review]

File: core/sass/components/_case_study.scss [hunk 1]
```diff
@@ -0,0 +1,17 @@
+.cast-study__lead_title {
+ margin-bottom:10px;
```
Comment (rafa-garcia) [inline core/sass/components/_case_study.scss:2]: These addes styles are mostly margin adjustments, we can do it using helper classes better (e.g. `margin-bottom: 10px` -> class `m-b-xs`)

File: core/templates/core/case_study_block.html [hunk 2]
```diff
@@ -5,10 +5,10 @@
 <div id="case_study">
     <div class="case-study p-t-m p-b-s">
         <div class="case-study__content media-block">
-            <i class="fas fa-newspaper"></i>
+            <i class="fas fa-quote-left"></i>
             <div>
-                <h3 class="h-m m-b-xs p-0">{{case_study.summary}}</h3>
-                <p class="case-study__company h-xs p-0">{{case_study.company_name}}</p>
+                <h3 class="cast-study__lead_title h-m m-b-xs p-0">{{case_study.lead_title}}</h3>
```
Comment (rafa-garcia) [inline core/templates/core/case_study_block.html:10]: We have redundant margin styles here, one in from the .css class `cast-study__lead_title` and the other the helper class `m-b-xs`

**Review by rafa-garcia — CHANGES_REQUESTED** [review]

File: core/sass/components/_case_study.scss [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+}
+.case-study__company {
+  color: #787191;
```
Comment (rafa-garcia) [inline core/sass/components/_case_study.scss:5]: There's also a helper class for this colour, `text-blue-deep-60`

File: core/sass/components/_case_study.scss (hunk 1 above)
Comment (webbyfox) [reply core/sass/components/_case_study.scss]: Good pick

File: core/sass/components/_case_study.scss (hunk 3 above)
Comment (webbyfox) [reply core/sass/components/_case_study.scss]: @rafa-garcia amended

File: core/sass/components/_case_study.scss [hunk 4]
```diff
@@ -0,0 +1,16 @@
+@import 'helpers/_colours';
```
Comment (bobmeredith) [inline core/sass/components/_case_study.scss:1]: Do we need this .css file at all.  It's all just margins and a colour that  can all be achieved by helper classes.
(although you have to be a bit lax on the pixel spacing as designers continue to use 10,20 etc where the helpers are 7.5, 15, 30

**Review by bobmeredith — APPROVED** [review]
LGTM

**Review by rafa-garcia — APPROVED** [review]
👍

**Pull request merged**